samples/client-sdk/knocking/index.js

```diff
@@ -417,7 +415,7 @@ const createGuestCall = async ({ name, url }) => {
   showLoadingText('guest');
 
   // Create call object
-  callObject = await window.DailyIframe.createCallObject();
+  callObject = window.DailyIframe.createCallObject();
```

Comment:
`createCallObject()` is not actually an async function (unless the heat in Gothenburg is really frying my brain right now, pls sanity check me >.<)